pkg/nodebootstrap/assets.go

```diff
@@ -86,7 +86,7 @@ func _10EkscltAl2Conf() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "10-eksclt.al2.conf", size: 999, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
+	info := bindataFileInfo{name: "10-eksclt.al2.conf", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
```

Comment:
Where do we use `asset.info`? I could only find `asset.bytes` being used but not `info`

Reply:
It was used It's no longer used for setting remote permissions, but we didn't make use of it in a long time, hence it's being killed in this PR as it made the build messy - as local permissions cannot be relied on... (well git track's owner's bit, but not group's or others').

Reply:
I see, thanks for the info. We should plan to remove having this field completely.

Reply:
Looks like go-bindata doesn't support making these fields optional.